src/pybind/mgr/cephadm/module.py

```diff
@@ -2474,6 +2482,35 @@ def _config_rgw(self, spec):
             'value': spec.rgw_realm,
         })
 
+        if spec.rgw_frontend_ssl_certificate:
+            if isinstance(spec.rgw_frontend_ssl_certificate, (list, tuple)):
```

Comment:
just wondering, how can this be a tuple?

Reply:
well a tuple being more of an immutable list, so really just depends on how the data was sent in. This just covers bases, so if it's any "list" like data structure we'll handle it.

Reply:
I'll remove the tuple so we can resolve this discussion, poke me if I forget ;)